component/client-handler/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/jwt/util/Util.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.util;
+
+import org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.SQLQueries;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.wso2.carbon.identity.application.authentication.framework.util.FrameworkUtils.isTableColumnExists;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.GET_JWT;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.GET_JWT_ID;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.GET_JWT_DETAILS;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.INSERT_JWD_ID;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.UPSERT_H2;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.UPSERT_MSSQL_DB2;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.UPSERT_MYSQL;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.UPSERT_ORACLE;
+import static org.wso2.carbon.identity.oauth2.token.handler.clientauth.jwt.Constants.UPSERT_POSTGRESQL;
+
+
+/**
+ * Util method for jwt token handler component.
+ */
+public class Util {
+
+    private static boolean isTenantIdColumnIsAvailableInIdnOidcAuthTable = false;
+    private static Map<String, String> queries = new HashMap<>();
+
+    public static boolean isTenantIdColumnAvailableInIdnOidcAuth() {
+
+        return isTenantIdColumnIsAvailableInIdnOidcAuthTable;
+    }
+
+    public static String getDBQuery(String key) {
+
+        return queries.get(key);
+    }
+
+    public static void checkIfTenantIdColumnIsAvailableInIdnOidcAuthTable() {
+
+        isTenantIdColumnIsAvailableInIdnOidcAuthTable = isTableColumnExists(SQLQueries.IDN_OIDC_JTI,
+                SQLQueries.TENANT_ID);
+        buildQueryMapping();
+    }
+
+    private static void buildQueryMapping() {
+
+        if (isTenantIdColumnIsAvailableInIdnOidcAuthTable) {
```

Comment:
Isn't these all values added to the `queries` map each time this method get executed (by checkIfTenantIdColumnIsAvailableInIdnOidcAuthTable).

Reply:
checkIfTenantIdColumnIsAvailableInIdnOidcAuthTable  will be called only once when the component is activated.